src/views/ProviderView/index.js

```diff
@@ -163,7 +163,8 @@ module.exports = class ProviderView {
         url: `${this.Provider.fileUrl(this.plugin.getItemRequestPath(file))}`,
         body: {
           fileId: this.plugin.getItemId(file)
-        }
+        },
+        provider: this.Provider
```

Comment:
This will most likely not work with GoldenRetriever, it needs state to be JSON.stringify-able in most cases

Reply:
And Redux Store too (which we support) I believe.

Reply:
Hasn’t changed, right?

Reply:
@arturi @goto-bus-stop fixed this now